### ISSUE TITLE
Embedded videos and content are not keyboard focusable

### ISSUE THREAD

**Issue by danielpost** [rqrauhvmra/Tobi#32]
When a lightbox is open, using the Tab key only focuses the Close and Prev/Next buttons. This is correct from an accessibility standpoint, however when embedding a video the user is unable to focus this video using their keyboard. This means that when autoplay is disabled keyboard user has no way to play an embedded video. I haven't tested it, but I suspect the same applies to links and buttons inside inline HTML.

I think the problem occurs [here](https://github.com/rqrauhvmra/Tobi/blob/master/js/tobi.js#L871).

PS. I'm about to open a few issues with bugs/feature requests since I figured it will be easier to keep track of items in separate issues. I'd be more than happy to contribute where I can. In another comment you mentioned working on a rewrite in a private repository so I didn't want to open any pull requests if you're also working on a rewrite.

**Comment by rqrauhvmra** [comment]
Hey @danielpost,

> when embedding a video the user is unable to focus this video using their keyboard [...] I suspect the same applies to links and buttons inside inline HTML

You're right, I didn't think about that 😞.

> In another comment you mentioned working on a rewrite in a private repository so I didn't want to open any pull requests if you're also working on a rewrite.

That's right 🎉, but all changes are implemented there as well. You are welcome to open a PR at any time on this repository.


Thanks for your feedback.

**Comment by danielpost** [comment]
I think [a11y-dialog](https://github.com/edenspiekermann/a11y-dialog/blob/master/a11y-dialog.js#L388) has a good implementation of trapping focus without missing anything, it might be worth to look into this.

I could submit a pull request here, but does it make sense if you're working on a rewrite? I feel like it would be twice the work, first me making the changes then you adapting them to fit into the new version. I think it would be better if I contribute any changes to the rewrite, if you're open to that.

**Comment by rqrauhvmra** [comment]
I'll take a look at it. Thanks.

PRs are always welcome 😃. It's no extra work for me and I learn more when I implement the code myself again but much more important: The release of Tobii (version 2) will take some time and won't replace this version.
